docs/source/models/causes/alzheimers/presymptomatic_and_mci_gbd_2021/index.rst

```diff
@@ -178,10 +184,251 @@ Cause Model Diagram
+      if we think the midpoint rule with a single rectangle gives a good
+      estimate of the area under the population curve).
+  * - deaths_c543
+    - Deaths from Alzheimer's disease and other dementias
+    - codcorrect
+    -
+  * - prevalence_c543
+    - Prevalence of Alzheimer's disease and other dementias
+    - como
+    -
+  * - incidence_rate_c543
+    - GBD's "total population incidence rate" for Alzheimer's disease
+      and other dementias
+    - como
+    - Raw GBD value, different from "susceptible incidence rate"
+      automatically calculated by Vivarium Inputs
+  * - acmr
+    - All-cause mortality rate
+    - loaded from ``_all.nc`` file provided by FHS Team
+    -
+  * - csmr_c543
+    - Cause-specific mortality rate for Alzheimer's disease and other
+      dementias
+    - :math:`\frac{\text{deaths_c543}}{(\text{population}) \cdot (\text{1 year})}`
+    - Calculated automatically by Vivarium Inputs
+  * - emr_c543
+    - Excess mortality rate for Alzheimer's disease and other dementias
+    - :math:`\frac{\text{csmr_c543}}{\text{prevalence_c543}}`
+    - Calculated automatically by Vivarium Inputs
+  * - emr_X
+    - Excess mortality rate in cause state X
+    - Values listed in state data table above
+    -
+  * - m_X
+    - Mortality hazard in cause state X
+    - acmr --- csmr_c543 + emr_X
+    -
+  * - sequelae_c543
+    - Sequelae of Alzheimer's disease and other dementias
+    - Set of 3 sequelae: s452, s453, s454
+    - Obtained from gbd_mapping.
+      Sequela names are "Mild," "Moderate," or "Severe Alzheimer's
+      disease and other dementias," respectively.
+  * - :math:`\text{prevalence}_s`
+    - Prevalence of sequela :math:`s`
+    - como
+    -
+  * - :math:`\text{DW}_s`
+    - Disability weight of sequela :math:`s`
+    - YLD Appendix
+    - For reference, the values are:
+
+      - s452: 0.069 (0.046-0.099)
+      - s453: 0.377 (0.252-0.508)
+      - s454: 0.449 (0.304-0.595)
+  * - :math:`\text{DW}_\text{c543}`
+    - Average disability weight of AD-dementia
+    - :math:`\sum_\limits{s\in \text{sequelae_c543}}
+      \text{DW}_s \cdot \text{prevalence}_s`
+    - Prevalence-weighted average disability weight over sequelae,
+      computed automatically by Vivarium Inputs. Used to calculate
+      YLDs.
+  * - :math:`\text{DW}_\text{MCI}`
+    - Disability weight of mild cognitive impairment
+    - ?
+    -
+  * - :math:`t`
+    - The current time in the simulation
+    - Changes with simulation clock
+    -
+  * - :math:`T_\text{BBBM}`
+    - The time at which the simulant enters the BBBM-AD state
+    - Random variable for each simulant
+    -
+  * - :math:`\alpha`, :math:`\lambda`
+    - Shape and rate parameters, respectively, of gamma distribution for
+      waiting time :math:`T` in BBBM-AD
+    - * :math:`\alpha = 468.75`
+      * :math:`\lambda = 125`
+    - Chosen so that :math:`P(3.5 < T < 4) \approx 0.9` because client
+      said, "The BBBM+ state lasts about 3.5--4 years before
```

Comment:
I wonder if they actually want something less concentrated than this; the parameters they have selected will imply a small effect for their treatment, and will make it relatively insensitive to *when* in the BBBM-AD period the treatment begins; perhaps they did not realize this.  A sensitivity analysis on this distribution could be something they value!